src/core/edge-resistance.c

```diff
@@ -285,7 +285,6 @@ find_nearest_position (const GArray        *edges,
           if (dist < best_dist)
             {
               best = compare;
-              best_dist = dist;
```

Comment:
Care to explain how this is unused?

Reply:
there is a break below

Reply:
Ah, I was looking at wrong line... Sorry!

Reply:
don't worry